src/ua/lviv/iot/algo/part1/lab2/Tricycle.java

```diff
@@ -0,0 +1,23 @@
+package ua.lviv.iot.algo.part1.lab2;
+
+import lombok.*;
```

Comment:
Do not use import *, as it can cause these problems:

Namespace pollution: Importing all names from a module can pollute your namespace with a large number of names, many of which you may not use. This can make your code harder to read and understand, especially for others who may need to maintain or modify your code.

Name collisions: If two modules define functions or variables with the same name, importing both modules with import * can lead to name collisions and unexpected behavior.

Code readability: Explicitly importing only the names you need can make your code more readable and easier to understand. It also makes it easier for others to see which modules and functions your code relies on.

Reply:
Intellij IDEA itself put *, but I forgot to fix it)